.github/workflows/ci.yml

```diff
@@ -45,16 +44,25 @@ jobs:
             ${{ runner.os }}-test-${{ env.cache-name }}-
             ${{ runner.os }}-test-
             ${{ runner.os }}-
+      - name: Install gcc-multilib
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.julia-arch == 'x86' }}
+        shell: bash
+        run: |
+          sudo apt install -y gcc-multilib
+      - name: Install i686-w64-mingw32-gcc
+        if: ${{ matrix.os == 'windows-latest' && matrix.julia-arch == 'x86' }}
+        shell: bash
+        run: |
+          choco install mingw --x86 --force --params "/exception:sjlj"
```

Comment:
Do we perhaps need to install `make` as well?  The error message makes me think that the `--force` might be removing the `make.exe` that comes with this image, and we're not re-installing it, perhaps?

Reply:
Maybe?  I have no idea how Chocolatey works, I just copied a comment from the Internet

Reply:
Ok, reinstalling `make` works :tada: